crates/primitives/src/l1.rs

```diff
@@ -21,6 +21,7 @@ use bitcoin::{
 use borsh::{BorshDeserialize, BorshSerialize};
 use rand::rngs::OsRng;
 use reth_primitives::revm_primitives::FixedBytes;
+use secp256k1::SECP256K1;
```

Comment:
I prefer to keep it as `bitcoin::secp256k1::SECP256K1` as before since it is easier to keep in-sync with `bitcoin`.
Can we revert this change?